specifyweb/specify/models.py

```diff
@@ -2815,6 +2819,7 @@ class Discipline(models.Model):
     geologictimeperiodtreedef = models.ForeignKey('GeologicTimePeriodTreeDef', db_column='GeologicTimePeriodTreeDefID', related_name='disciplines', null=False, on_delete=protect_with_blockers)
     lithostrattreedef = models.ForeignKey('LithoStratTreeDef', db_column='LithoStratTreeDefID', related_name='disciplines', null=True, on_delete=protect_with_blockers)
     modifiedbyagent = models.ForeignKey('Agent', db_column='ModifiedByAgentID', related_name='+', null=True, on_delete=protect_with_blockers)
+    taxontreedef = models.ForeignKey('TaxonTreeDef', db_column='TaxonTreeDefID', related_name='discipline', null=True, on_delete=protect_with_blockers)
```

Comment:
Sorry, I know this is a WIP. But, don't we need the reverse of that? 

We don't want multiple disciplines being able to share a tree. We need multiple trees being able to exist in a single discipline. You can't do that without a FK in taxontreedef table OR a middle join table. Or am I misunderstanding something?